modules/nw-dns-operator.adoc

```diff
@@ -39,7 +39,7 @@ $ oc get clusteroperator/dns
 [source,terminal]
 ----
 NAME      VERSION     AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
-dns       4.1.15-0.11  True        False         False      92m
+dns       4.16        True        False         False      92m
```

Comment:
When I tested this on a live cluster I got:
`dns    4.17.0-0.nightly-2024-07-29-061317   True        False         False      21m `
I think we could just change the 4.17.0 to 4.16.z. I'll ping QE to double check.

Reply:
@JoeAldinger , this can be because you are running 4.17.0 nightly release.
We can make the change like 4.16.0